macros/public/get_chain_explorer_link.sql

```diff
@@ -0,0 +1,22 @@
+{% macro get_chain_explorer_link() %}
```

Comment:
Actually @viniabussafi I think this exists already - https://github.com/duneanalytics/spellbook/blob/9c360192574f965f7f2e90c618efd259affba8df/macros/public/get_chain_explorer_address.sql#L1 .

Can you take a look and if it's the same thing indeed can you update the PR to just add the `get_balancer_link macro`?

Reply:
hey, @antonio-mendes! The only difference at this point is that get_chain_explorer_address returns an URL but not a proper link to the chain explorer
![image](https://github.com/duneanalytics/spellbook/assets/131974393/e5044c8e-5fc4-4a03-ae06-6da5fb55e42f)

Reply:
The returning of a link is nice, my only suggestions is to use `https://etherscan.io/search?f=0&q=` as the base link and build off of that. The link I posted will return the proper page for a block, token/eoa, and tx. Just double check that it works for the chains you have listed.

I would also suggest changing the name to `get_link` or something. The shorter the better.

Reply:
Agree the scope is different and having a clickable link is useful; but maybe we can refactor the code to reuse the existing macro @viniabussafi ? Also, I think you're missing the closing `</a>` at the end of each record.

Reply:
I would also suggest making the link text an argument of the function for increased flexibility and renaming the arguments for readability

`get_chain_explorer_link(chain_ STRING, address_ STRING, link_text_ STRING)`

Reply:
Actually nvm, just noticed `get_href` exists, so this function's purpose can be achieved with 

`get_href(get_chain_explorer_address(blockchain, address), link_text)`

Reply:
Ah ok I get the point now. My question is more it can be confusing to have:
1. `get_chain_explorer_link`
2. `get_chain_explorer_address` (already in Prod)
3. `get_chain_explorer_tx_hash` (already in Prod)

Knowing that 2 and 3 are a subset of 1. But also if we just remove 2 and 3 to use just 1 we break existing queries.

I like the idea of a general `get_chain_explorer_link` that is clickable and intelligent enough to adapt, so the ideal think would be to have this macro, and maybe rework 2 and 3 so it "redirects" to 1 in the background? What do you think?

Reply:
I agree, @antonio-mendes. Will work on them and tag you once they´re done. Thanks a lot

Reply:
> maybe rework 2 and 3 so it "redirects" to 1 in the background
Maybe I misunderstand, but I don't think we can change 2 and 3 because queries that use them expect them to return the URL and nothing more.

Reply:
Markus is right. Maybe we can just stick with `get_href` atm and only have get_balancer_link right now.

Reply:
Makes sense to me! We can then think about improving existing macros going forward in individual PRs